client/dist/style.css

```diff
@@ -0,0 +1,92 @@
+#avgstar{
+  font-size:50px;
+}
+.star{
+  vertical-align: top;
+}
+#recommendrate{
+  margin-bottom: 10px;
+}
+.stars5{
+  border: 1px solid;
+  border-radius: 2px;
+  padding-right: 100px;
+  box-shadow: none;
+  margin-left: 10px;
+  background-color: darkgreen;
+}
+
+.greybar{
+  display: inline-block;
+  width: 200px;
+  background-color: darkgray;
+  margin-left: 4px;
+}
+.greenbar{
+  background-color: green;
+}
+
+.characteristic{
+  margin-top: 10px;
+  width: 250px;
+}
+.left{
```

Comment:
try to use more specific names for your class

Reply:
will change it.